springer-fachzeitschriften-medizin-psychologie.csl

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Springer Fachzeitschriften Medizin Psychologie (German)</title>
+    <id>http://www.zotero.org/styles/springer-fachzeitschriften-medizin-psychologie</id>
+    <link href="http://www.zotero.org/styles/springer-fachzeitschriften-medizin-psychologie" rel="self"/>
+    <link href="http://www.zotero.org/styles/springer-basic-brackets-no-et-al-alphabetical" rel="template"/>
+    <link href="http://static.springer.com/sgw/documents/135255/application/pdf/Musterbeitrag_Uebersichten_Leitthema_Schwerpunkt.pdf" rel="documentation"/>
+    <link href="http://static.springer.com/sgw/documents/69331/application/pdf/Leitfaden_Uebersichten_Leitthemen_Schwerpunkt.pdf" rel="documentation"/>
+    <author>
+      <name>Philipp Zumstein</name>
+      <uri>https://github.com/zuphilip</uri>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <category field="psychology"/>
+    <summary>Springer Basic style (German, numeric, brackets, alphabetical) for Springer journals in medicine and psychology (linked to EndNote_Literaturstyle_SpringerFachzeitschriftenMedizinPsychologie_StandDez2012.zip).</summary>
```

Comment:
please remove the EndNote reference. I understand why it's there, but given the legal history, I'd rather not refer to their styles at all.